### DISCUSSION TITLE
budibase opensource vs enterprise

### DISCUSSION THREAD

**Discussion by unknown1337** [Budibase/budibase#4568] [General]
Dear,

I like budibase and I'm wondering if in future that there is a chance that users need to go to a paid plan because some features are there though not supported in the opensource version? (apart from SLA/ support, hosting, auditing and version control which are part of the enterprise package?). 

Thus, will there be a part of the budibase core that is not solely needed for the hosting of the enterprise environment that might not be opensource available later?

Thanks!

**Comment by mjashanks** [comment]
Everything that is in the product today is open-source, and will remain that way forever.

But yes - there will be future features that will not be open-source. As a rule of thumb, features that an individual developer uses to build an app, will be open source.

Features that are used to manage Budibase deployments with lots of users and apps, will be paid. Examples of future paid features:
- User grouping
- Global datasources
- Global design system
- Auditing
- Environments

These are features that will make your life a lot easier, if you have lots of users, lots of apps - or compliance needs.

**Reply by unknown1337** [reply]
thanks for your quick answer!
